src/context/VerifyContext.js

```diff
@@ -102,26 +102,26 @@ export function VerifyProvider(props) {
 
   // EVENT HANDLERS
 
-  const approve = async ({ approveAction, id }) => {
+  const approve = async ({ approveAction, captureImage }) => {
     if (!approveAction) {
       throw Error('no approve action object!');
     }
     if (approveAction.isApproved) {
       log.debug('approve');
       await api.approveCaptureImage(
```

Comment:
These inputs no longer tally with the updated function signature.

Reply:
Do you mean I should update the naming of approveAction and captureImage?  I updated from id where approve is invoked.  Verify seems to refer to the capture object as captureImage throughout.  Sorry, I don't understand what change is needed.

Reply:
Here you're calling:
```
approveCaptureImage(capture, morphology, age, captureApprovalTag, speciesId)
```
but the function signature in `treeTrackerApi.js` is:
```
approveCaptureImage(capture, morphology, speciesId, captureApprovalTag)
```

I'd suggest reverting the one in `treeTrackerApi.js` to get `age` back in.

Reply:
Thank you for the explanation.  I was looking at the wrong function.  Will do.